bugbug/similarity.py

```diff
@@ -163,6 +164,9 @@ def evaluation(self):
     def get_distance(self, query1, query2):
         return
 
+    def load(self):
```

Comment:
Make this a static method, otherwise users will have to instantiate an object first just to load another object.